working_with_health_tool/README.md

```diff
@@ -1,31 +1,41 @@
-# Generate a Ballerina Package from Health Tool and Develop a FHIR API
+# Generate a Ballerina Package using the Health Tool and develop an FHIR API
 
-Using the Ballerina-Health Tool, you can generate a Balllerina package
-for a given FHIR Implementation Guide.
-This guide shows how to generate a package and use the generated package in a REST API
+You can generate a Ballerina package using the Ballerina-Health Tool for a given FHIR implementation guide.
+
+This guide shows how to generate a package and use that generated package in a REST API.
+
+## Set up the pre-requisites
 
-### Pre-requisites
 - Ballerina [Swan Lake Update 7](https://ballerina.io/downloads/) or higher
-- Ballerina Health Tool installed
+- Ballerina Health Tool installed by executing the command below.
   > `bal tool pull health`
 
-### Generate Ballerina Package using Health Tool
+## Generate the Ballerina package using the Health Tool
+
+1. Clone the repository to your local machine.
+
+2. Navigate to the `working_with_health_tool` directory.
+
+  > `cd working_with_health_tool`
+
```

Comment:
doesn't make sense as you are already there.

Reply:
both 1 & 2 are unnecessary IMO.

Reply:
The first step is required as we need to instruct the users to clone the repo.

The second step is also required as the repo can be cloned to any preferred location whereas, the package generation command has to be executed after navigating to the cloned repo.